packages/block-editor/src/components/inner-blocks/README.md

```diff
@@ -125,6 +125,7 @@ Template locking of `InnerBlocks` is similar to [Custom Post Type templates lock
 Template locking allows locking the `InnerBlocks` area for the current template.
 _Options:_
 
```

Comment:
I've tested inheritance and this is how it works:

- if a CPT [defines](https://github.com/WordPress/gutenberg/blob/trunk/docs/reference-guides/block-api/block-templates.md#locking) its `template_lock` to `noContent` will have all "container blocks" (group, cover, column so far) to behave like if all of them have the `templateLock: noContent` applied in their attributes. This is expected and how other values for templateLock work.
- if a individual block defines `templateLock: noContent`, unlike any other locks, no children can override it.

Reply:
We forgot to document the new state for Custom Post Types. https://github.com/WordPress/gutenberg/pull/43977